vhdbuilder/packer/buildperformance/evaluate-build-performance.sh

```diff
@@ -56,5 +55,11 @@ for script in "${scripts[@]}"; do
   echo "##[endgroup]"
 done
 
-rm ${SIG_IMAGE_NAME}-build-performance.json
+mv ${SIG_IMAGE_NAME}-build-performance.json vhdbuilder/packer/buildperformance
+pushd vhdbuilder/packer/buildperformance
+  echo -e "\nRunning build performance evaluation program...\n"
+  ./buildPerformance
```

Comment:
Just wanted to confirm, this would fail the pipeline step if the binary panics right?

Reply:
Thanks for pointing this out! I originally chose panic because I read that log.Fatalf would not allow the deferred functions for closing client, ctx, etc to execute. I will make sure the panics are handled gracefully and the step doesn't fail.

Reply:
I see, I wasn't sure if we intended to fail the step on error.